workers/unity/Packages/io.improbable.gdk.testutils/MockCommandSender.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.Commands;
+using Improbable.Gdk.TestUtils;
+using Unity.Entities;
+
+namespace Packages.io.improbable.gdk.testutils
+{
+    public class MockCommandSender : ICoreCommandSender
+    {
+        private readonly MockWorld world;
+        public long NextRequestId { get; private set; } = 1;
+
+        private readonly Dictionary<Type, uint> componentIds = new Dictionary<Type, uint>();
+
+        private readonly Dictionary<Type, HashSet<long>> requestIds = new Dictionary<Type, HashSet<long>>();
+
+        private readonly Dictionary<long, ICommandRequest> outboundRequests = new Dictionary<long, ICommandRequest>();
+
+        internal MockCommandSender(MockWorld world)
+        {
+            this.world = world;
+        }
+
+        public void Setup<TRequest>(uint componentId)
```

Comment:
I could probably get rid of this setup function using 
`componentId = (uint?) declaringType?.GetField("ComponentId", BindingFlags.Public | BindingFlags.Static)?.GetValue(null);`